### DISCUSSION TITLE
flink-doris-connector并没有攒批或者间隔时间写入数据

### DISCUSSION THREAD

**Discussion by X1304951263** [apache/doris#25286] [A - General / Q&A] [answered]
`properties.setProperty("sink.batch.size","20000");
        properties.setProperty("sink.batch.interval","3000");

        DorisExecutionOptions.Builder  executionBuilder = new DorisExecutionOptions.Builder();
        executionBuilder.setBufferCount(20000)
                .setCheckInterval(3000)
                .setMaxRetries(3)
                .setDeletable(false)
                .setStreamLoadProp(properties);`

这是我的代码，按照官网的代码示例写的，但是经过测试，dorisSink  并没有攒批写入数据或者间隔3秒写入数据，而是  一次性将源表的1000W条数据 一次性写入doris  尝试了很多配置都没有用，  flink 1.17.1版本，flink-doris-connector 1.4.0版本

**Comment by JNSimba** [comment, marked as answer]
This is the previous version. The batch writing has been reconstructed on the latest master. You can refer to this PR https://github.com/apache/doris-flink-connector/pull/168
